multimodal/src/autogluon/multimodal/data/process_mmlab/process_mmdet.py

```diff
@@ -0,0 +1,100 @@
+import logging
+from typing import Dict, List, Optional
+
+import numpy as np
+import PIL
+from PIL import ImageFile
+from torch import nn
+
+try:
+    from torchvision.transforms import InterpolationMode
+
+    BICUBIC = InterpolationMode.BICUBIC
+except ImportError:
+    BICUBIC = PIL.Image.BICUBIC
+
+from ...constants import AUTOMM
+from ..utils import is_rois_input
+from .process_mmlab_base import MMLabProcessor
+
+logger = logging.getLogger(AUTOMM)
+ImageFile.LOAD_TRUNCATED_IMAGES = True
+
+
+class MMDetProcessor(MMLabProcessor):
```

Comment:
It seems that `MMDetProcessor ` has the same implementation as `MMOcrProcessor`. Why do we need to define both?

Reply:
Currently our implementation for MMOCR only includes simple inference, and it temporarily reuses the logic from MMDet. Later they should be different